docs/Architecture.md

```diff
@@ -76,7 +76,7 @@ _Benchmark runners_ output 3 files into the folder specified by the _global runn
 
 ### Global Runner
 
-_Global runner_ is a script that consequitively runs all benchmarks using corresponding _benchmark runners_ while enforcing specified hard time limits.
+_Global runner_ is a script that consecutively runs all benchmarks using corresponding _benchmark runners_ while enforcing specified hard time limits.
```

Comment:
One day, this might run benchmarks in parallel, so suggest simply to delete "consecutively" -- it is not necessary to understand this methodology?

Reply:
Yes, consecutiveness is indeed not necessary.